content/community/links.html

```diff
@@ -22,20 +20,17 @@
 <li><a href="http://www.besly.de/" target=_blank>BeSly</a>: German BeOS, Haiku and ZETA knowledge base. Tutorials, translations 
 and more in several languages.</li>
 <li><a href="http://betips.net/" target=_blank>BeTips</a>: The Haiku/BeOS Tip Server.</li>
-<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source  projects. <a href="https://haikuarchives.github.io/software.html">Software Archives <a></li>
+<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source projects.</li>
+<li><a href="https://haikuarchives.github.io/software.html" target=_blank>HaikuArchives Index</a>: A partial index of the Haiku Archives.</li>
```

Comment:
Is it useful to link this? No one is maintaining it and there isn't any useful info there besides what's already in the github?

Reply:
Removed.